.github/workflows/reusable-workflow.yml

```diff
@@ -89,13 +100,25 @@ jobs:
           CYPRESS_BIDDER_MNEMONIC_INPUT: ${{ inputs.bidder_mnemonic }}
           CYPRESS_BIDDER_ADDRESS_INPUT: ${{ inputs.bidder_address }}
 
-      - name: Archive e2e artifacts
+      - name: Archive e2e artifacts locally
```

Comment:
_low priority_

I don't think `locally` is accurate. I'd call it `Upload e2e artifacts as workflow artifacts` or something similar

Reply:
Makes sense. Updated